admin/admin-filters-sanitize.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * Setup specific admin filters usefull for sanitization.
+ *
+ * Extract from PLL_Admin_Filters to be able to use in a REST API context.
```

Comment:
This should rather be in the PR's comment. Actually, this class is also used in a non REST API context.

Reply:
Maybe it wasn't clear but the code already existed before in PLL_Admin_Filters which is used in classic editor context.
One part of PLL_Admin_Filters_Sanitize class extract the code to sanitize title and username and be able to instantiate it in the two contexts.